api/app/Enums/ExecCoaching.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Enums;
+
+use App\Traits\HasLocalization;
+
+enum ExecCoaching
+{
+    use HasLocalization;
+
+    case COACHING;
+    case LEARNING;
+
```

Comment:
**Exec coaching status**
- Not participating
- Have a coach
- Is a coach
- Both coaching/learning

same question as Mentorship status below 
Are we not storing  Not participating as sep enum value?

Reply:
I envisioned it being an array.

- `[]` = not participating
- `null` = not provided

Reply:
that works as well!